tee-worker/ts-tests/integration-tests/examples/direct-invocation/index.ts

```diff
@@ -16,17 +16,19 @@ import {
 import { getEnclave, sleep, buildIdentityHelper, initIntegrationTestContext } from '../../common/utils';
 import { aesKey, keyNonce } from '../../common/call';
 import { Metadata, TypeRegistry } from '@polkadot/types';
-import sidechainMetaData from '../../litentry-sidechain-metadata.json';
+import sidechainMetaData from '../../litentry-sidechain-metadata.json' assert { type: 'json' };
```

Comment:
`sidechain-api` should be reexporting this as a const instead

Reply:
I haven't made any changes because even after multiple modifications, I am still encountering the `ERR_PACKAGE_PATH_NOT_EXPORTED` error. Considering that this is a difficult problem to debug, let's continue using it as it is for now. :(

Reply:
Please open a followup ticket so we don't forget to revisit this at a later point :)

Reply:
https://github.com/litentry/litentry-parachain/issues/1881